Formula/mrbayes.rb

```diff
@@ -26,7 +26,20 @@ class Mrbayes < Formula
   depends_on "open-mpi"
 
   def install
-    system "./configure", *std_configure_args, "--with-mpi=yes"
+    args = ["--with-mpi=yes"]
+    if Hardware::CPU.intel? && build.bottle?
+      args << "--disable-avx"
+      # There is no argument to override AX_EXT SIMD auto-detection, which is done in
+      # configure and adds -m<simd> to build flags and also defines HAVE_<simd> macros
+      args << "ax_cv_have_sse41_cpu_ext=no" unless MacOS.version.requires_sse41?
+      args << "ax_cv_have_sse42_cpu_ext=no" unless MacOS.version.requires_sse42?
+      args << "ax_cv_have_sse4a_cpu_ext=no"
+      args << "ax_cv_have_sha_cpu_ext=no"
+      args << "ax_cv_have_aes_cpu_ext=no"
+      args << "ax_cv_have_avx_os_support_ext=no"
+      args << "ax_cv_have_avx512_os_support_ext=no"
```

Comment:
Monterey logs (`*_cpu_ext` are printed out as cached while `*_os_support_ext` force disables and causes warning):
```
checking whether SSE4.2 is supported by the processor... yes
checking whether SSE4.2 is supported by the processor and OS... yes
checking whether C compiler accepts -msse4.2... yes
checking whether SSE4a is supported by the processor... (cached) no
checking whether SHA is supported by the processor... (cached) no
checking whether AES is supported by the processor... (cached) no
checking whether AVX is supported by the processor... yes
checking whether AVX is supported by the processor and OS... no
configure: WARNING: Your processor supports AVX, but your OS doesn't
checking whether FMA3 is supported by the processor... no
checking whether FMA4 is supported by the processor... no
checking whether XOP is supported by the processor... no
checking whether AVX2 is supported by the processor... no
checking whether AVX512-F is supported by the processor... no
checking whether AVX512-CD is supported by the processor... no
checking whether AVX512-PF is supported by the processor... no
checking whether AVX512-ER is supported by the processor... no
checking whether AVX512-VL is supported by the processor... no
checking whether AVX512-BW is supported by the processor... no
checking whether AVX512-DQ is supported by the processor... no
checking whether AVX512-IFMA is supported by the processor... no
checking whether AVX512-VBMI is supported by the processor... no
```